docs/API.TraversingOutboundAndInboundLinksWithXgAndYg.md

```diff
@@ -349,12 +359,21 @@ There are ways to address this using **yg** and SPARQL queries, but they have li
 
 The **gqp** flag is intended to handle this requirement. Here is an example of a search clause that uses **gqp** for the scenario described above:
 
-    <cm-well-host>/?op=search&qp=type.rdf:Person,age>32&gqp=<addressOfPerson>physicalAddress[city::NY]
+    <cm-well-host>/?op=search&qp=type.rdf:Person,age>32&gqp=<addressOfPerson>physicalAddress[city::New%20York]
 
 The **gqp** flag's syntax is identical to the **yg** flag's syntax, but it operates differently. As **yg** traverses the links defined in its value expression, it adds all the infotons in the link paths to its results. **gqp** does not add infotons to the results in the set returned by the search query. Rather it attempts to evaluate its expression for each "root" infoton, and if it fails at some stage in the evaluation or the expression evaluates as **false**, then the root infoton from which the path originated is *removed* from the result set.
 
 The **gqp** flag can be used together with **xg** and **yg**. In this case, **gqp** takes precedence, meaning that first results are filtered by the **gqp** expression, and then expanded by **xg** and **yg**.
 
 >**Notes:** 
 >* The **gqp** flag can be applied to both **consume** and **search** operations. Note that when using **gqp** with consume, it is possible to filter out the entire chunk and therefore to receive no results for some iterations. If in this case you receive a 204 error, but the position token in the header is different than the one sent, you still need to keep consuming.
->* The [ghost skips](#hdr13) behavior applies to **qgp** as well as **yg**.
\ No newline at end of file
+>* The [ghost skips](#hdr13) behavior applies to **qgp** as well as **yg**.
+
+<a name="hdr16"></a>
+## Implementing Multiple Filter Expressions with the Pipe Operator ##
+
+As for the **yg** query, you can add several filter expressions to a single **gqp** query, separated by the pipe ('|') operator. However, the behavior of the complex expression is different than for **yg**. When piped expressions are used with **gqp**, CM-Well applies "or" logic among them. This means that in order to match the filter, an infoton only needs to match *one* of the piped expressions.
+
+For example, the following query retrieves infotons of people who live *either* in New York *or* in Chicago.
+
+    <cm-well-host>/?op=search&qp=type.rdf:Person&gqp=<addressOfPerson>physicalAddress[city::New%20York]|<addressOfPerson>physicalAddress[city::Chicago]
```

Comment:
Again, not very good example as the filter could be `*city::Chicago,*city::New%20York`, which is preferable.

Reply:
OK, I'm cool with the yg example, now could you please provide a good example for pipes in gqp?

Reply:
e.g: for the president example, let's say I want to search all the US presidents that either were born outside the US, or were living in the white house before 2000.  
`gqp` would be:
```
gqp=<hasTenant.xmpl[hasBeginning.time<2000-01-01]>location.locn[name.geonames::The White House]|>birthPlace.madsrdf[-countryCode::US]
```